sanitize-html/sanitize-html.d.ts

```diff
@@ -1,27 +1,65 @@
-﻿// Type definitions for sanitize-html 1.6.0
+﻿// Type definitions for sanitize-html 1.12.0
 // Project: https://github.com/punkave/sanitize-html
-// Definitions by: Rogier Schouten <https://github.com/rogierschouten>
```

Comment:
please do not remove the current author name, add to the list. it is a comma separated list.